iis-wf/iis-wf-preprocessing/src/main/resources/eu/dnetlib/iis/wf/preprocessing/main/oozie_app/workflow.xml

```diff
@@ -213,6 +213,11 @@
 			<value>/user/${user.name}/iis/working_dirs/${execution_environment}</value>
 			<description>working directory</description>
 		</property>
+        <property>
+            <name>reports_external_path</name>
+            <value>$UNDEFINED$</value>
```

Comment:
Default value is quite crucial. It will be good not to require `reports_external_path` to be set if someone is not interested in gathering reports in external location.

Thus we should either provide here default location (e.g. `${workingDir}/report/report.json`) where report will be copied if external location is not desired.

Other approach could rely on `$UNDEFINED$` which is already set. But we would need to handle it properly in subworkflow handling copying report - e.g. by skipping copying when `reports_external_path=$UNDEFINED$`. Currently this is not the case, `$UNDEFINED$` directory will be created.